packages/ui/src/kit/VInputGroup/VInputGroup.theme-default.scss

```diff
@@ -0,0 +1,31 @@
+.v-input-group.theme-default {
+  display: flex;
+  justify-content: stretch;
```

Comment:
Add `align-items` property too

Reply:
It's `flex-start` by default. Do you want just make it more explicit, or guess that we should set something different, not `flex-start`?